drink_machines.lua

```diff
@@ -397,6 +417,9 @@ minetest.register_node('drinks:liquid_barrel', {
       if outputstack == 'vessels:glass_bottle' then
          drinks.drinks_liquid_sub(4, 'jbo', 128, pos)
       end
+      if outputstack == 'vessels:steel_bottle' then
```

Comment:
Well, those if-chains just happen when something starts small and grows more and more. 
I didn't wanted to change the structure too much, but adding the 5th condition I agree it's somehow time for a table ;-).
I would suggest also to combine drinks.lua and drink2.lua, as they share mostly the same content.